File: c/Lista3/Desafio8.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
    
    float dias, meses, anos, IdadeEmDia;
    
printf("Digite anos: ");
scanf("%f", &anos);
    
printf("Digite meses: ");
scanf("%f", &meses);
    
printf("Digite dias: ");
scanf("%f", &dias);
    
IdadeEmDia = (anos * 365) + (dias * 30) + dias;
printf("A idade em dias é: %3.0f", IdadeEmDia);
    
} 

File: js/lista3/Desafio8.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio8- Faça um algoritmo que leia a idade de uma pessoa expressa em anos, meses e dias mostre-a expressa apenas em dias:

Ignore variações de meses como fevereiro.*/

rl.question('Digite a quantidade de anos: ', (anos) => {
    rl.question('Digite a quantidade de meses: ', (meses) => {
        rl.question('Digite a quantidade de dias: ', (dias) => {

        let idadeEmDias = (anos * 365) + (meses * 30) + ( dias * 1);
    
        console.log(`A idade da pessoa em dias é: ${idadeEmDias} dias`);
      rl.close();
        });
    });
 });